docs/BR.md

```diff
@@ -1008,6 +1065,66 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
+
+The "# of Network Perspectives and Quorum Requirements" Table describes quorum requirements related to Multi-Perspective Issuance Corroboration. 
+
+CAs MAY immediately retry Multi-Perspective Issuance Corroboration using the same validation method or an alternative method (e.g., a CA can immediately retry validation using "Email to DNS TXT Contact" if "Agreed-Upon Change to Website - ACME" does not corroborate the outcome of Multi-Perspective Issuance Corroboration). There is no stipulation regarding the maximum number of validation attempts that may be performed in any period of time.
+
+Phased Implementation Timeline:
+- *Effective March 15, 2024*, the CA SHOULD implement Multi-Perspective Issuance Corroboration as described in this section. 
+- *Effective September 15, 2024*, the CA MUST implement Multi-Perspective Issuance Corroboration as described in this section. However, the CA MAY proceed with certificate issuance if the number of additional Network Perspectives which do not corroborate the primary determinations ("non-corroborations") is greater than the number allowed in the "Quorum Requirements" table. The CA MUST attempt to corroborate the primary determinations from at least two (2) additional Network Perspectives.
+- *Effective March 15, 2025*, the CA MUST NOT proceed with certificate issuance if the number of additional Network Perspectives which do not corroborate the primary determinations is greater than the number allowed in the "Quorum Requirements" table.
+- *Effective September 15, 2025*, the CA MUST attempt to corroborate the primary determinations from at least three (3) additional Network Perspectives. Additionally, the Network Perspectives that do corroborate the primary determinations MUST fall within the service regions of at least two (2) distinct regional Internet registries.
+- *Effective March 15, 2026*, the CA MUST attempt to corroborate the primary determinations from at least four (4) additional Network Perspectives. 
+- *Effective September 15, 2026*, the CA MUST attempt to corroborate the primary determinations from at least five (5) additional Network Perspectives. 
+
+Table: Quorum Requirements
+
+| # Network Perspectives Used | # of Allowed non-Corroborations |
+| --- | --- |
+| 2-5 |  1  |
+| 6+  |  2  |
+
+Network Perspectives performing Multi-Perspective Issuance Corroboration:
+
+- MUST:
+    - Forward all Internet traffic via a network or set of networks that filter all RPKI-invalid BGP routes as defined by RFC 6811. 
+
+- SHOULD:
+    - Facility & Service Provider Requirements
+      - Be hosted from an ISO/IEC 27001 certified facility or equivalent. 
+      - Rely on services covered in one of the following reports: System and Organization Controls 3 (SOC 3), IASE 3000, ENISA 715, FedRAMP Moderate, C5:2020, CSA STAR CCM, or equivalent.
```

Comment:
Same comment as above: "...or equivalent **services framework independently audited and certified or reported**."

SOC 3 is rarely used as a compliance requirement. To receive a SOC 3 report an organization must undergo a SOC 2 audit. I suggest we just use SOC 2 here.

Reply:
Good suggestions, will incorporate.

Reply:
Addressed in https://github.com/ryancdickson/staging/commit/de1ff61609a027bfd0cf5183285a1a4d80da7e4a